[DM-1294]: Added 3 announcements for device grid changes. (#241)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/announcements-10-17-0.md

As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), with release 10.18+, the "Map" widget will be migrated and support for real-time updates on all devices will be replaced by a configurable refresh interval. This change allows us to display more then 100 devices on the map by default. Additionally, the "Map" widget will no longer support the "show track" option.

> ##### Deprecated device-grid model classes, column implementations and services

Reviewer: ```suggestion
##### Deprecation of device-grid model classes, column implementations and services
```

> Shared classes, components and services defined in @c8y/ngx-components/device-grid are now moved to @c8y/ngx-components. Therefore the obsolete items are deprecated.

Reviewer: ```suggestion
Shared classes, components and services previously defined in @c8y/ngx-components/device-grid have now been moved to @c8y/ngx-components. Therefore the obsolete items are deprecated.
```

> This change only affects you, if you or your development team use the Web SDK to extend Cumulocity IoT UI applications or to build your own web applications. If you do and you do use the device-grid functionalities, check the deprecation documentation and alter your code accordingly. Refer to the deprecations in the WebSDK resources documentation for the device-grid service [here](http://resources.cumulocity.com/documentation/websdk/ngx-components/injectables/DeviceGridService.html). Other deprecations for reference are also marked in this documentation.

Reviewer: ```suggestion
This change only affects you, if you or your development team use the Web SDK to extend Cumulocity IoT UI applications or to build your own web applications. If you use the device-grid functionalities, check the deprecation documentation and alter your code accordingly. Refer to the deprecations in the [WebSDK resources documentation for the device-grid service](http://resources.cumulocity.com/documentation/websdk/ngx-components/injectables/DeviceGridService.html). Other deprecations for reference are also marked in this documentation.
```
